scripts/zkai_d64_nested_verifier_backend_spike_gate.py

```
@@ -0,0 +1,732 @@
+        seen_pairs.add(pair)
+        case_pairs.append(pair)
+        if case["baseline_result"] != RESULT:
+            raise D64NestedVerifierBackendSpikeError(f"mutation case {index} baseline_result mismatch")
+        if not isinstance(case["mutated_accepted"], bool):
+            raise D64NestedVerifierBackendSpikeError(f"mutation case {index} mutated_accepted must be boolean")
+        if not isinstance(case["rejected"], bool):
+            raise D64NestedVerifierBackendSpikeError(f"mutation case {index} rejected must be boolean")
+        if case["rejected"] == case["mutated_accepted"]:
+            raise D64NestedVerifierBackendSpikeError(f"mutation case {index} rejected/accepted fields are inconsistent")
+        if not isinstance(case["rejection_layer"], str) or not case["rejection_layer"]:
+            raise D64NestedVerifierBackendSpikeError(f"mutation case {index} rejection_layer must be a non-empty string")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)